CantEngine/Components/MeshComponent.cpp

```diff
@@ -28,7 +28,8 @@ RTTR_REGISTRATION
 }
 
 MeshComponent::MeshComponent(GameObject *owner)
-	:BaseComponent(owner, static_type)
+	:BaseComponent(owner, static_type),
+	m_mesh_type(MESH_MODEL)
```

Comment:
Is this to fix a bug?

Reply:
I noticed at some point a meshcomponent has garbage value for mesh type, so I thought this will make it not have garbage value